packages/service/db/migrations/20210726224018_role-auth.sql

```diff
@@ -0,0 +1,27 @@
+-- migrate:up
+ALTER TABLE document ENABLE ROW LEVEL SECURITY;
+
+CREATE FUNCTION current_user_id() RETURNS uuid AS $$
+  SELECT nullif(current_setting('app.user_id', true), '')::uuid;
+$$ LANGUAGE SQL STABLE;
+
+CREATE ROLE postgraphile_user;
```

Comment:
CREATE ROLE may require elevated privileges, and is disabled on some shared cloud db instances. Check to make sure GCP SQL instance will let you do this.